homeassistant/components/risco/binary_sensor.py

```diff
@@ -61,85 +45,34 @@ async def async_setup_entry(
         )
 
 
-class RiscoBinarySensor(BinarySensorEntity):
-    """Representation of a Risco zone as a binary sensor."""
+class RiscoCloudBinarySensor(BinarySensorEntity, RiscoCloudZoneEntity):
```

Comment:
I'd invert the inheritance order, so that the integration specific classes are closest to the child.